src/main/java/stellarium/DrawSky.java

```diff
@@ -321,11 +326,12 @@ else if (mc.theWorld.provider.isSurfaceWorld())
 					double longdd=(double)longcd/(double)longn;
 					double latdd=1.0-(double)(latc+1)/(double)latn;
 
-					tessellator1.setColorRGBA_F(1.0f, 1.0f, 1.0f, (f4*moonilum[longc][latc]-0.15f*bglight)*2.0f);
+					float lightlevel = (0.875f*(bglight/2.1333334f));
+					tessellator1.setColorRGBA_F(1.0f - lightlevel, 1.0f - lightlevel, 1.0f - lightlevel, ((f4*moonilum[longc][latc]-0.015f*bglight)*2.0f));
```

Comment:
Why apply 1.0f - lightlevel?
and if you want this, just multiply this to the alpha value.